.stylelintrc

```diff
@@ -0,0 +1,103 @@
+		"function-disallowed-list": null,
+		"function-linear-gradient-no-nonstandard-direction": true,
+		"function-name-case": "lower",
+		"function-url-no-scheme-relative": true,
+		"function-url-quotes": "always",
+		"function-url-scheme-allowed-list": null,
+		"function-url-scheme-disallowed-list": null,
+		"hue-degree-notation": "angle",
+		"keyframe-declaration-no-important": true,
+		"keyframes-name-pattern": "[a-z][-a-z0-9]+",
+		"length-zero-no-unit": [true, { "ignore": ["custom-properties"] }],
+		"max-nesting-depth": 0,
+		"media-feature-name-allowed-list": null,
+		"media-feature-name-disallowed-list": null,
+		"media-feature-name-no-unknown": true,
+		"media-feature-name-no-vendor-prefix": true,
+		"media-feature-name-value-allowed-list": null,
+		"no-descending-specificity": null,
+		"no-duplicate-at-import-rules": true,
+		"no-duplicate-selectors": true,
+		"no-empty-source": true,
+		"no-invalid-double-slash-comments": true,
+		"no-unknown-animations": true,
+		"number-max-precision": 8,
+		"property-allowed-list": null,
+		"property-disallowed-list": null,
+		"property-no-unknown": [true, { "checkPrefixed": true, "ignoreProperties": ["app-region", "accent-color"] }],
+		"property-no-vendor-prefix": [true, { "ignoreProperties": ["box-orient", "mask-image", "appearance"] }],
+		"rule-empty-line-before": ["always-multi-line", { "except": ["first-nested"] }],
+		"selector-attribute-operator-allowed-list": null,
+		"selector-attribute-operator-disallowed-list": null,
+		"selector-attribute-quotes": "always",
+		"selector-class-pattern": "[a-z][-a-z0-9]+",
+		"selector-combinator-allowed-list": null,
+		"selector-combinator-disallowed-list": null,
+		"selector-id-pattern": "[a-z][-a-z0-9]+",
+		"selector-max-attribute": 4,
+		"selector-max-class": 8,
+		"selector-max-combinators": 4,
+		"selector-max-compound-selectors": 4,
+		"selector-max-id": 2,
+		"selector-max-pseudo-class": 4,
+		"selector-max-specificity": "4,4,4",
+		"selector-max-type": 4,
+		"selector-max-universal": 2,
+		"selector-nested-pattern": null,
+		"selector-no-qualifying-type": null,
+		"selector-no-vendor-prefix": true,
+		"selector-pseudo-class-allowed-list": null,
+		"selector-pseudo-class-disallowed-list": null,
+		"selector-pseudo-class-no-unknown": true,
+		"selector-pseudo-element-allowed-list": null,
+		"selector-pseudo-element-colon-notation": "double",
+		"selector-pseudo-element-disallowed-list": null,
+		"selector-pseudo-element-no-unknown": true,
+		"selector-type-case": "lower",
+		"selector-type-no-unknown": [true, { "ignore": ["custom-elements"] }],
+		"shorthand-property-no-redundant-values": true,
+		"string-no-newline": true,
+		"time-min-milliseconds": 120,
+		"unit-allowed-list": null,
+		"unit-disallowed-list": null,
+		"unit-no-unknown": true,
+		"value-keyword-case": ["lower", { "camelCaseSvgKeywords": true }],
+		"value-no-vendor-prefix": [true, { "ignoreValues": ["box", "grab", "no-grab"] }]
+	}
+}
```

Comment:
I'm usually not a fan of excessive lint rules but I imagine it's strict because you're generating and saving files

Reply:
We can remove stylelint for sure. It is a linter for CSS to have a more "normalized" code. Most of the rules are not too restrictive though, like, there is no rule for order of properties or things like that.

I'm just a weird person that doesn't like other people's configs and I like to be explicit in all of the options, so lots of them are turned off though. If it is better, we can cleanup the rules to be more concise or use someone else's config, no problem.